Updates the README with instructions how to get the example app running

MaxGabriel opened the pull request:
As a prospective user of Subliminal, it was important to me to check out the example app and see how things worked. Unfortunately, running the example app wasn't exactly turnkey. 

Absent it being easier to setup, I added instructions to the README about how to get it working. Since the errors you run into without installing the submodules or running the Rake task don't prompt obvious fixes, I think having instructions is important. 

I wasn't sure about where to place the instructions; I ended up putting them towards the end and linking to them from the "Getting Started" section (unrelated, I also fixed the link from Getting Started to the FAQ)

Very excited about Subliminal :+1: Sorry I was kind of out of it when I asked you the web view question on Wednesday -- coffee had worn off by that point

MaxGabriel committed:
Update README with Example app instructions

MaxGabriel committed:
Update README to fix internal links

The links from "How to Get Started" to the FAQ and the Example app instructions were broken

MaxGabriel committed:
Update README to capitalize App

wearhere commented:
Hey, thanks Max! Very cool to get our first 3rd-party pull request, especially when it's fixing our bugs! :wink: I've just a few comments.

MaxGabriel committed:
Update README per PR comments

1. Remove git submodule instructions
2. Move the "Running the Example App" instructions to right after "How to Get Started"

I noticed that you only need to run `rake install` once per computer, so I also prefixed that step with "If you haven't already"

wearhere commented:
Thanks again, @MaxGabriel!
